x-pack/plugins/security_solution/public/management/cypress/tasks/agent_policy.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { API_VERSIONS, agentPolicyRouteService, agentRouteService } from '@kbn/fleet-plugin/common';
+import { uninstallTokensRouteService } from '@kbn/fleet-plugin/common/services/routes';
+import type { GetUninstallTokensMetadataResponse } from '@kbn/fleet-plugin/common/types/rest_spec/uninstall_token';
+import type { UninstallToken } from '@kbn/fleet-plugin/common/types/models/uninstall_token';
+import { request } from './common';
+
+export const enableAgentTamperProtectionFeatureFlagInPolicy = (agentPolicyId: string) => {
+  return request({
+    method: 'PUT',
+    url: agentPolicyRouteService.getUpdatePath(agentPolicyId),
+    body: {
+      name: 'With agent tamper protection enabled',
```

Comment:
Why update the name of the policy? (same for `description` below and perhaps others?)

Reply:
just to describe that this policy has agent tamper protection enabled. I'm ok with keeping the original name/description if not